.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark issues as stale
```

Comment:
kev told me not to include a pr staler on main.

Reply:
>stale_issues.yml
But yeah noted